src/main/java/sirius/web/controller/BasicController.java

```diff
@@ -172,4 +174,61 @@ public void onError(WebContext ctx, HandledException error) {
         }
         defaultRoute.accept(ctx);
     }
+
+    @Override
+    public void onJsonError(WebContext ctx, HandledException error) {
```

Comment:
Bitte beim nächsten PR Logik-Verschiebungen von Logik-Änderungen in Commits trennen, das hilft ungemein beim Review weil so wie es jetzt hier ist das git diffing 0 hilfreich sein kann.